Lets Do This/Views/Reportback/LDTReportbackItemDetailView.m

```diff
@@ -129,4 +129,39 @@ - (IBAction)shareButtonTouchUpInside:(id)sender {
     }
 }
 
+- (void)setPreferredMaxLayoutWidthForLabels {
+    CGFloat width = CGRectGetWidth([UIScreen mainScreen].bounds);
+    NSLog(@"setPreferredMaxLayoutWidthForLabels width %f", width);
+    self.reportbackItemCaptionLabel.preferredMaxLayoutWidth = width - 16;
+    self.reportbackItemQuantityLabel.preferredMaxLayoutWidth = width / 3;
+    self.userCountryNameLabel.preferredMaxLayoutWidth = 100;
+}
+
+- (CGSize)preferredLayoutSizeFittingSize:(CGSize)targetSize {
+    CGRect originalFrame = self.frame;
+
+    // step1: set the detailView.frame to use our target width
+    self.frame = CGRectMake(self.frame.origin.x, self.frame.origin.y, targetSize.width, targetSize.height);
+    NSLog(@"targetSize.width %f", targetSize.width);
+
+    // step2: layout the cell
```

Comment:
Also tried adding  
````
[self setNeedsUpdateConstraints];
[self updateConstraintsIfNeeded];
````
per http://stackoverflow.com/q/28670951/1470725 but no luck. [This article](http://devetc.org/code/2014/07/07/auto-layout-and-views-that-wrap.html), which is at the bottom of the SO answer i'm following under related reading --  suggests using `layoutSubviews`, which we did do in #480 but not here yet.  Although that article suggests calling `[super layoutSubviews]` again after you call it once, which i haven't yet tried in #687